[14.0][ADD] project_forecast_line_milestone

ntsirintanis opened the pull request.

dessanhemrayev committed:
[ADD] project_task_description_template: new module

ntsirintanis commented on project_forecast_line_milestone/models/project_task.py:
```diff
@@ -0,0 +1,26 @@
+            if this.date_deadline:
+                this.forecast_date_planned_end = this.date_deadline
+                continue
+            this.forecast_date_planned_end = this.forecast_date_planned_end
```
This appears to be possible in 14.0

gjotten replied:
@ntsirintanis yup seems to go quite well!

ntsirintanis commented on project_forecast_line_milestone/models/project_task.py:
```diff
@@ -0,0 +1,31 @@
+    def _compute_forecast_date_planned_end(self):
+        """Override method to use milestone_id.target_date"""
+        res = super()._compute_forecast_date_planned_end()
+        for this in self:
```
change this to task in self

ntsirintanis committed:
[14.0][ADD] project_forecast_line_milestone

jarmokortetjarvi committed:
Added translation using Weblate (Finnish)

jarmokortetjarvi committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (21 of 21 strings)

Translation: project-14.0/project-14.0-project_category
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_category/fi/

jarmokortetjarvi committed:
Added translation using Weblate (Finnish)

jarmokortetjarvi committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (8 of 8 strings)

Translation: project-14.0/project-14.0-project_deadline
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_deadline/fi/

jarmokortetjarvi committed:
Added translation using Weblate (Finnish)

jarmokortetjarvi committed:
Translated using Weblate (Finnish)

Currently translated at 100.0% (8 of 8 strings)

Translation: project-14.0/project-14.0-project_tag
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_tag/fi/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (40 of 40 strings)

Translation: project-14.0/project-14.0-project_recalculate
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_recalculate/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (94 of 94 strings)

Translation: project-14.0/project-14.0-project_forecast_line
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_forecast_line/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: project-14.0/project-14.0-project_task_recurring_activity
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_recurring_activity/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (76 of 76 strings)

Translation: project-14.0/project-14.0-project_role
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_role/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translation: project-14.0/project-14.0-project_mail_chatter
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_mail_chatter/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: project-14.0/project-14.0-project_task_project_required
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_project_required/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: project-14.0/project-14.0-project_key
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_key/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (59 of 59 strings)

Translation: project-14.0/project-14.0-project_wbs
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_wbs/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: project-14.0/project-14.0-project_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_purchase_link/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: project-14.0/project-14.0-project_stock_request
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_stock_request/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: project-14.0/project-14.0-project_task_digitized_signature
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_digitized_signature/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (12 of 12 strings)

Translation: project-14.0/project-14.0-project_sequence
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_sequence/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: project-14.0/project-14.0-project_task_add_very_high
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_add_very_high/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 87.5% (7 of 8 strings)

Translation: project-14.0/project-14.0-project_task_send_by_mail
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_send_by_mail/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (78 of 78 strings)

Translation: project-14.0/project-14.0-project_timesheet_time_control
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_timesheet_time_control/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (44 of 44 strings)

Translation: project-14.0/project-14.0-project_gtd
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_gtd/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: project-14.0/project-14.0-project_tag
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_tag/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: project-14.0/project-14.0-project_duplicate_subtask
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_duplicate_subtask/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translation: project-14.0/project-14.0-project_task_default_stage
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_default_stage/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (23 of 23 strings)

Translation: project-14.0/project-14.0-project_task_dependency
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_dependency/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translation: project-14.0/project-14.0-project_hr
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_hr/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: project-14.0/project-14.0-project_forecast_line_deadline
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_forecast_line_deadline/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: project-14.0/project-14.0-project_parent_task_filter
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_parent_task_filter/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: project-14.0/project-14.0-project_task_code
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_code/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (57 of 57 strings)

Translation: project-14.0/project-14.0-project_stock
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_stock/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: project-14.0/project-14.0-project_deadline
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_deadline/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: project-14.0/project-14.0-project_template
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_template/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (29 of 29 strings)

Translation: project-14.0/project-14.0-project_milestone
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_milestone/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (26 of 26 strings)

Translation: project-14.0/project-14.0-project_status
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_status/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (21 of 21 strings)

Translation: project-14.0/project-14.0-project_category
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_category/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: project-14.0/project-14.0-project_task_pull_request
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_pull_request/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: project-14.0/project-14.0-project_timesheet_time_control_sale
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_timesheet_time_control_sale/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: project-14.0/project-14.0-project_template_milestone
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_template_milestone/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: project-14.0/project-14.0-project_stage_state
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_stage_state/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Translation: project-14.0/project-14.0-project_stage_mgmt
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_stage_mgmt/es/

OCA-git-bot committed:
Merge PR #1192 into 14.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update project_task_description_template.pot

OCA-git-bot committed:
[BOT] post-merge updates

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: project-14.0/project-14.0-project_task_description_template
Translate-URL: https://translation.odoo-community.org/projects/project-14-0/project-14-0-project_task_description_template/it/

OCA-git-bot committed:
Merge PR #1248 into 14.0

Signed-off-by dreispt

Commit:
[UPD] Update project_forecast_line_priority.pot

OCA-git-bot committed:
[BOT] post-merge updates

thomaspaulb commented:
@ntsirintanis This non-OCA MR can probably be closed, no?

ntsirintanis commented:
> @ntsirintanis This non-OCA MR can probably be closed, no?

yes, it is merged now
